.github/workflows/upgrade_automtion.yml

```diff
@@ -0,0 +1,18 @@
+name: Upgrade Automation
```

Comment:
Do you want to keep this file named the same or will you follow up with an update to this? https://github.com/flyteorg/boilerplate/blob/master/.github/workflows/master.yaml#L19

Reply:
Now we have only one workflow for upgrade all component like flyteidl, boilerplate etc